superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -18,12 +18,15 @@ const propTypes = {
   controls: PropTypes.object.isRequired,
   forcedHeight: PropTypes.string,
   form_data: PropTypes.object.isRequired,
+  metrics: PropTypes.number.isRequired,
   standalone: PropTypes.bool.isRequired,
   triggerQuery: PropTypes.bool.isRequired,
+  queryResponse: PropTypes.object,
```

Comment:
You shouldn't have to touch this file at all. queryResponse will be available in your visualization context in `nvd3.jsx`